source/Ship.cpp

```diff
@@ -2720,6 +2738,25 @@ void Ship::WasCaptured(const shared_ptr<Ship> &capturer)
 
 
 
+// This ship was ejected as an escape pod. Inherit characteristics of the ejector.
+void Ship::WasEjected(const shared_ptr<Ship> &ejector)
+{
+	government = ejector->GetGovernment();
+	// If the ejecting ship had a parent, use that as the escape pod's parent.
+	SetParent(ejector->GetParent());
+	// This ship behaviors like its ejector did.
+	isSpecial = ejector->isSpecial;
+	isYours = ejector->isYours;
+	personality = ejector->personality;
+	
+	// Escape pods won't get targeted for the first 3 seconds after being ejected.
+	// Escape pods are very fragile ships, and this gives them a chance to escape.
+	if(attributes.Category() == "Escape Pod")
```

Comment:
I prefer not to put special meaning to categories here, especially if we also want Fighters and Drones to be possible candidates to be used as escape pods.
Alternatives is to put the "untargetable" property on escape pods directly, or maybe even add an "untargetable after escape launch" attribute to each ship that can be used as escape pod.

Reply:
I'd prefer to go with the attribute route instead of having all escape pods be untargetable.